live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/mapper/MimeTypeMapper.java

```diff
@@ -0,0 +1,77 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.live_ingester.adapters.messaging.repository.mapper;
+
+import java.util.Map;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import org.alfresco.hxi_connector.live_ingester.adapters.config.IntegrationProperties;
+
+@Component
+@RequiredArgsConstructor
+public class MimeTypeMapper
+{
+
+    public static final String EMPTY_MIME_TYPE = "";
+    static final String DEFAULT_MIME_TYPE = "application/pdf";
+    private final IntegrationProperties integrationProperties;
+
+    public String mapMimeType(String inputType)
+    {
+        Map<String, String> mappings = MapUtils.emptyIfNull(integrationProperties.alfresco().transform().mimeType().mapping());
```

Comment:
`.mimeType().mapping()` shouldn't be with 's' at the end (in plural form)?

Reply:
I honestly don't know how to tackle that as there seem not to be solid conventions for properties keys naming (at least that I know of). 
So the question here is whether to configure the key with singular (as it is now):
``` yml
    mime-type:
      mapping:
        '[image/png]': "image/png"
        ....
```
or with plural:
``` yml
    mime-type:
      mappings:
        '[image/png]': "image/png"
        ....
```
I am okay with any. @tpage-alfresco @Pawel-608 WDYT?

Reply:
IMO plural indicates, that there can be more then one value (or pair), similarly to List or Set

Reply:
Mapping or mappings are both fine from an English pov. A "mapping" could be a single source-target pair, or it could be the entire instruction set.

For the filter you could change it to `allows` and `denies`, but IMO it reads as a verb rather than a plural.

Reply:
Maybe `aspects`, `types` and `paths` should be in plural instead of `allows` and `denys` 🤔

Reply:
I think this is fine tbh. Customers are going to need to consult the docs anyway, so I don't think there's a huge value in changing this.